tests/test_tasks.py

```diff
@@ -44,17 +44,32 @@ def testIndexGeotiffIndexesFromFGDC(job, bag):
 
 def testSubmitToDspaceUploadsSwordPackage(job, bag_tif):
     with patch('kepler.tasks.sword.submit') as mock:
+        job.item.handle = ""
```

Comment:
Having to set this here and on line 54 makes me concerned the code may not do what is intended. That said, prior to overriding this value it seemed to be set to a mocked value.

Is it better to set it to `None` during this test instead of `""` and update the check here to `if job.item.handle == None:`:
https://github.com/MITLibraries/kepler/pull/74/files#diff-6a049d02522de3377f8de13c8e5ab88aR70

Mostly I'm just confused why `job.item.handle` had a value before it was being set during the tests.